### ISSUE TITLE
[feature request] repeat alt+letter to jump to the previous focused tab

### ISSUE THREAD

**Issue by eoli3n** [osa1/tiny#362]
I like that feature of sway, and it would be great to be able to use it with tiny.

```
workspace_auto_back_and_forth yes|no
When yes, repeating a workspace switch command will switch back to the prior workspace. For example, if you are currently on workspace 1, switch to workspace 2, then invoke the workspace 2 command again, you will be returned to workspace 1. Default is no.
```

**Comment by osa1** [comment]
Sounds like a good idea to me :+1:

**Comment by Kabouik** [comment]
I like that feature in Sway too but am not sure how to reconcile it with the current behaviour in `tiny` when there are multiple channels using the same letter, which inevitably happens when many channels are listed. Currently, pressing `Alt+letter` will cycle through those channels. Having letters automatically set for all channels is convenient too, so disabling that to avoid duplicate keybinds would be frustrating.

Would it work if `/switch` without a suffix brings back to the last used channel, and gets a keybind?

**Comment by osa1** [comment]
Right, I think we want to make this work with `alt+number` combinations, not with `alt+letter`.
